runtime/src/lib.rs

```diff
@@ -278,7 +278,7 @@ impl pallet_scheduler::Config for Runtime {
 	type WeightInfo = ();
 }
 
-pub const BLOCKS_PER_DAY: BlockNumber = 24 * 60 * 10;
+pub const BLOCKS_PER_DAY: BlockNumber = 10;
 
```

Comment:
Is this just for testing this issue more quickly? (Just want to make sure this doesn't get merged in... :) )

Reply:
good catch! leftovers from another test